lib/Ray/sources/Window.hpp

```diff
@@ -90,7 +90,7 @@ namespace RAY {
 
 			//! @brief Set background color (framebuffer clear color)
 			//! @param color The color to clear the screen (default: black)
-			void clear(const Color &color = BLACK);
+			void clear(Color color = BLACK);
```

Comment:
Why isn't the parameter a const reference ?

Reply:
Because you can"t put lvalue in const references parameters but you always use it like that (c macro that initialiez a class is a lvalue)